### ISSUE TITLE
Docs: calcjob metadata.option are difficult to find

### ISSUE THREAD

**Issue by ramirezfranciscof** [aiidateam/aiida-core#5139]
### Describe the current issue

Helping @flavianojs submit his first calculation to an external cluster, we found it not trivial to find how to set some of the `metadata.options` such as `account`, `mpirun_extra_params` or `custom_scheduler_commands`. Initially we went into "How-To Guides" -> "How to run external calculations" -> ["How to submit a calculation"](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/run_codes.html#how-to-submit-a-calculation). There we found the following example but not much else on other options or other resources:

```
builder.metadata.options.resources = {
    'num_machines': 1,
    'num_mpiprocs_per_machine': 1,
}
```

We were able to find [this](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/calculations/usage.html?highlight=mpirun_extra_params#options) but only after I checked some of my calculations and we specifically "Searched the docs" for the keywords `mpirun_extra_params`.

### Describe the solution you'd like

There should probably be (easy to detect, like in an admonition) links to the [options](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/calculations/usage.html?highlight=mpirun_extra_params#options) and the [resources](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/schedulers.html) sections from either the [how-to submit](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/run_codes.html#how-to-submit-a-calculation), the [topics on launching](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/processes/usage.html#topics-processes-usage-launching) or (maybe better) both.

**Comment by ltalirz** [comment]
I second this

**Comment by MikeW1121** [comment]
Hey, I would love to work on this issue!

**Comment by ltalirz** [comment]
Hey @MikeW1121 

my apologies, I just realized that I fixed this in 82484f94573f33592a37a962fad97105dcf61ac7 2 days ago and did not remember there was an open issue for it.

Now there's a not for the pointer right below the section

![image](https://user-images.githubusercontent.com/1053245/222990472-d1df3f60-4eff-4b76-93ae-8339699dfba8.png)

I'm sure you'll find another issue worth fixing :-)
